seurat-find-variable-genes.R

```diff
@@ -73,6 +101,38 @@ option_list = list(
     default = NA,
     type = 'character',
     help = "File name in which to store variable genes in plain text."
+  ),
+  make_option(
+    c("--loess-span"),
+    action = "store",
+    default = 0.3,
+    metavar = "Loess span parameter for VST",
+    type = 'double',
+    help = "(vst method) Loess span parameter used when fitting the variance-mean relationship. Default: 0.3"
+  ),
+  make_option(
+    c("--clip-max"),
+    action = "store",
+    default = "auto",
```

Comment:
This one was tricky, but given that the default is a character.... although type should probably be a double.